platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -29,10 +29,10 @@
             type="String"
             default="/=karaf"/>
 
-        <AD description="List of authentication types required for each context. List of default valid authentication types are: SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
+        <AD description="List of authentication types required for each context. List of default valid authentication types are: IDP, SAML, BASIC, PKI, GUEST. Example: /context=AUTH1|AUTH2|AUTH3"
             name="Authentication Types" id="authenticationTypes" required="true" cardinality="100"
             type="String"
-            default="/=SAML|GUEST,/admin=SAML|basic,/system=basic,/solr=SAML|PKI|basic,/sources=SAML|basic,/security-config=SAML|basic"/>
+            default="/=IDP,/solr=SAML|PKI|BASIC"/>
```

Comment:
I looked in the docs and there aren't any places to update. We have example configurations in the documentation, but those should remain.